azure_functions_worker/testutils_lc.py

```diff
@@ -129,12 +129,24 @@ def _find_latest_mesh_image(cls,
                                f' Status {response.status_code}')
 
         tag_list = response.json().get('tags', [])
+        # Removing images with a -upgrade
```

Comment:
Better comment to describe why we do this.

Reply:
Fixed